08_voting/index.tex

```diff
@@ -1 +1,192 @@
+
+\textbf{Changes of rules:} the President can select a rule from the proposed list to be voted by the Voter islands. The Speaker will host the voting for changes of rules and be responsible for the ballots counting and results declaration. The ballots from the Voter islands will be: Aye, namely agree with the rule change; Nay, namely disagree with the rule change; or Abstain.
+
+
+\section{Voting Methods}
+\label{sec:VotingMethods}
+
+Table \ref{table:votingmethod} indicates available choices for each voting scenario.
+
+\begin{table}[H]
+\caption{Available voting methods for each voting scenario}\label{table:votingmethod}
+\begin{center}
+\begin{tabular}{ |p{3cm}||p{3cm}|p{3cm}|  }
+ \hline
+ Voting Methods   & Elections of Roles & Changes of Rules   \\
+ \hline
+ Plurality   &     & \checkmark     \\
+ \hline
+ Borda Count &  \checkmark   &      \\
+ \hline
+ Runoff      &  \checkmark    &        \\
+ \hline
+ Instant Runoff    & \checkmark  &     \\
+ \hline
+ Approval  &  \checkmark    &    \\
+ 
+ \hline
+\end{tabular}    
+\end{center}
+\end{table}
+
+\textbf{Plurality} will select the majority choices as the collective decision. When the decision is picking between two options, e.g. whether agree or disagree a change of rule. Unlike the elections, it is not necessary to compare the options left for determining a relative preference. It is straightforward and effective especially when the number of valid ballots (i.e. abstention excluded) is odd. However, it requires consideration when the results are tied. The solution to tied results is regimented with the assumption that the proposer of the voting in such events will vote for "agree". Thus, the proposal will be approved if the number of "agree" votes is more than the number of "disagree" votes. 
+
+\textbf{Borda Count} method will allow voters to rank preference instead of choosing one option only. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all voters' ballots produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the collective decision.
```

Comment:
suggested re-writing:

**\textbf{Borda Count} method **will allow Voters to rank their** preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate.** Thus, the candidate with the highest Borda score will be **the winner.** 



note that I write ''Voters'' as I defined this thing in the beginning of the section!

Reply:
changed